requirements/install.txt

```diff
@@ -1,3 +1,3 @@
-Jinja2==2.7.3
-PyYAML==3.11
-jsonschema==2.4.0
\ No newline at end of file
+Jinja2>=2.7.3
```

Comment:
Jinja 2.7.3 was released in June 2014, and they’ve been rather good at maintaining compatibility since then. No new 2.7.x was released since then.

I believe I tested with the latest version when I wrote this patch, and it did indeed work.